astropy/time/core.py

```diff
@@ -201,19 +204,6 @@ def _construct_from_dict_base(self, map):
 
         return out
 
-    def _construct_from_dict(self, map):
```

Comment:
This was just moved into `TimeInfo` verbatim except adding special handling for `datetime64` and `datetime` formats.